packages/engine/src/scene/components/ParticleSystemComponent.ts

```diff
@@ -73,7 +73,7 @@ import {
   useHookstate
 } from '@ir-engine/hyperflux'
 import { NameComponent } from '@ir-engine/spatial/src/common/NameComponent'
-import { addObjectToGroup, removeObjectFromGroup } from '@ir-engine/spatial/src/renderer/components/GroupComponent'
+import { addObjectToGroup, removeObjectFromGroup } from '@ir-engine/spatial/src/renderer/components/ObjectComponent'
```

Comment:
We'll probably need a different solution for the particle system component since the particle system library relies on the three hierarchy, so particle systems are not showing up on this branch

Reply:
dang, we can do the same thing as XRUI where we manually update the hierarchy in a system

Reply:
particles dont seem to be working on dev, so i'm not sure what the correct fix is here